src/Command/ExecuteCoreUpgradeScriptsCommand.php

```diff
@@ -104,14 +104,14 @@ protected function execute(InputInterface $input, OutputInterface $output){
 				$fsHelper->move($oldSourcesDir . '/apps', $oldSourcesDir . '/__apps');
 				$fsHelper->mkdir($oldSourcesDir . '/apps');
 				$plain = $this->occRunner->run('upgrade');
-				$fsHelper->removeIfExists($oldSourcesDir . '/apps');
-				$fsHelper->move($oldSourcesDir . '/__apps', $oldSourcesDir . '/apps');
```

Comment:
@Phiber2000 it would be better to leave these lines and copy them right after the line 110 (catch) because new exception is thrown in L112. The code after the catch clause will be never executed in this case